taskcat/stacker.py

```diff
@@ -1045,6 +1057,22 @@ def generate_input_param_values(self, s_parms, region):
                         print("{}Getting license content for {}/{}".format(D, license_bucket, licensekey))
                     _parameters['ParameterValue'] = param_value
 
+                if presignedurl_re.search(param_value):
+                    paramsplit = self.regxfind(presignedurl_re, param_value).split(',')[1:]
```

Comment:
Assumes that it has been used correctly and will raise if not. Perhaps trap cases where Len() < 2, print a meaningful error and exit(1)